### PR TITLE
Deprecate low use models

### PR DESCRIPTION
**Pull request by amyeroberts**
Deprecates models which are at least a year old and have had fewer than 5,000 downloads in the past 30 days
- graphormer
- xlm_prophetnet
- qdqbert
- nat
- ernie_m
- tvlt
- nezha
- mega
- jukebox
- vit_hybrid
- deta
- speech_to_text_2
- efficientformer
- realm
- gptsan_japanese

Left out of this (despite qualifying): 
* conditional detr - to give time to see effects of object detection script 
* time series transformer - to give time for checkpoints to be created
* x-clip was not added. It was picked up by the script run >month ago, but there are models currently with a large number of downloads in the past 30 days.

### CODE REVIEW

**Comment by qubvel** [comment]
Amy, Conditional DETR is now in [object detection guide](https://huggingface.co/docs/transformers/main/en/tasks/object_detection). It would be great to see its usage after a month before we will deprecate it. Actually, from my short experience training these models, Conditional DETR could be a drop-in replacement for DETR, because it's as fast as DETR, but converges faster.

**Comment by amyeroberts** [comment]
@qubvel OK, sounds good! I've removed Conditional DETR for now. We can see if the usage picks up over the next few months now the training script is available.

File: docs/source/en/model_doc/deta.md [hunk 1]
```diff
@@ -16,6 +16,14 @@ rendered properly in your Markdown viewer.
 
 # DETA
 
+<Tip warning={true}>
+
+This model is in maintenance mode only, we don't accept any new PRs changing its code.
+If you run into any issues running this model, please reinstall the last version that supported this model: v4.40.2.
+You can do so by running the following command: `pip install -U transformers==4.40.2`.
+
+</Tip>
```
[the remark this answers is not in the document]
Comment (ArthurZucker) [reply docs/source/en/model_doc/deta.md:25]: also would mention that they are no longer tested!

[the remark this answers is not in the document]
Comment (ArthurZucker) [reply docs/source/en/model_doc/deta.md:25]: and that you should still be able to use it

[the remark this answers is not in the document]
Comment (ArthurZucker) [reply docs/source/en/model_doc/deta.md:25]: and maybe add the commit that deprecated it for our reference?

**Review by ArthurZucker — COMMENTED** [review]
💔 my baby jukebox, I'll miss you!

**Review by ArthurZucker — APPROVED** [review]

File: docs/source/en/model_doc/deta.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (amyeroberts) [reply docs/source/en/model_doc/deta.md:25]: Would it be OK if I added that in a follow-up? This is the same message used for other models we currently have deprecated. This way I can update the script + all the other deprecated models without cluttering up this one.

[the remark this answers is not in the document]
Comment (ArthurZucker) [reply docs/source/en/model_doc/deta.md:25]: for sure!

**Comment by SangbumChoi** [comment]
@amyeroberts 

after merge this following error keep shows up in make fix-copies. Do you have any clue?

```
root@0fcd247a20e3:/mnt/nas2/users/sbchoi/transformers# make fix-copies
python utils/check_copies.py --fix_and_overwrite
Traceback (most recent call last):
  File "/mnt/nas2/users/sbchoi/transformers/utils/check_copies.py", line 1106, in <module>
    check_copies(args.fix_and_overwrite, args.file)
  File "/mnt/nas2/users/sbchoi/transformers/utils/check_copies.py", line 852, in check_copies
    new_diffs = is_copy_consistent(filename, overwrite, buffer)
  File "/mnt/nas2/users/sbchoi/transformers/utils/check_copies.py", line 675, in is_copy_consistent
    target_lines, theoretical_code, theoretical_code_splits = find_code_and_splits(
  File "/mnt/nas2/users/sbchoi/transformers/utils/check_copies.py", line 521, in find_code_and_splits
    code, (lines, target_start_index, target_end_index) = find_code_in_transformers(
  File "/mnt/nas2/users/sbchoi/transformers/utils/check_copies.py", line 435, in find_code_in_transformers
    raise ValueError(
ValueError: `object_name` should begin with the name of a module of transformers but got models.deta.modeling_deta.load_cuda_kernels.
make: *** [Makefile:83: fix-copies] Error 1
```

**Pull request merged**